docs/reference/index.md

```diff
@@ -117,14 +116,14 @@ parse the response and resolve it to any listening onResponse functions with the
 <a name="Connect+send"></a>
 
 ### connect.send(request, id, [opts])
-Send a request message to a uPort client.
+Send a request message to a uPort client. Useful function if you want to pass additional transport options and/or when send a request you already created elsewhere.
```

Comment:
Small simplification of sentence: "Useful function if you want to pass additional transport options or send a request you already created elsewhere."

Maybe get @shanshokr to do a review of the writing style for these docs?

Reply:
Yeah definitely, but I think it can be done after the release as well